packages/webapp/src/components/Navigation/useSectionHeaders.tsx

```diff
@@ -48,7 +48,7 @@ export function useSectionHeader(path: Pathname): string | React.ReactElement |
     <>
       {t('SECTION_HEADER.ANIMALS_INVENTORY')}
       <Badge
-        style={{ marginLeft: '10px' }}
+        className="positionRight"
```

Comment:
Since the styles are contained in the child component instead of being part of the parent, here I'd instead pass a "position" property set to either left or right, and then in the `Badge` component I'd set the appropriate class names doing something like `className = styles[position]` with .left and .right classes added to styles, or `className = position === 'left' ? styles.positionLeft : stylesPositionRight`.

Usually when a component accepts a `className` prop as shown here, it means that the parent can pass a className pointing to styles specified in the parent itself, so basically the parent can "customize" the way the child component looks. If we attempted to do this with the way this is set up it wouldn't work because Badge is looking for this class name in its own styles file. Hopefully that makes sense! Not a blocker for merging though.

Reply:
Initially, i coded it that way, not sure why i went with more complex solution. I was trying to handle two use case(menu-item and positioning) with one parameter. Thanks for correcting me though.